### PR TITLE
Add a Bulgarian translation for the 1.22 release blog post

### PR DESCRIPTION
**Pull request by ipanamski**

### CODE REVIEW

**Comment by yetist** [comment]
Build failed, I think you should change `conf.py` to add `bg` in `TRANSLATIONS` like this:
```
TRANSLATIONS = {
    DEFAULT_LANG: "",
    "bg": "./bg",
    "ca": "./ca",
    "de": "./de",
...
```

**Comment by raveit65** [comment]
https://github.com/mate-desktop/mate-desktop.org/tree/preview-143 created for preview your PR at https://preview.mate-desktop.dev/ in a while.

File: conf.py [hunk 1]
```diff
@@ -156,6 +157,15 @@
 		('/team/', 'Team'),
 		('/donate/', 'Donate'),
     ),
+    "bg": (
+		('/bg/blog/', 'Блог'),
+		('/bg/install/', 'Инсталация'),
+		('/bg/gallery/1.22/catalan/', 'Екранни снимки'),
```
Comment (raveit65) [inline conf.py]: Why Catalan languages for screenshots?
Might be better to use the english (default) screenshots.

Comment (ipanamski) [reply conf.py]: Yeah, i just copy pasted the Catalan one and didn't see that. For now lets use the english one, later I will upload a bulgarian screenshot

**Comment by raveit65** [comment]
Yes, bulgarian screenshot you can upload with another PR.

**Comment by raveit65** [comment]
rebased and squashed some commits befor merge.
https://github.com/mate-desktop/mate-desktop.org/commit/dbf5b3ce0e5a42e9dbf359adb7597f7f2287e19f
https://github.com/mate-desktop/mate-desktop.org/commit/bb2a27d98f2f54e24bab32d1b1eb708cbb2e391e
Thanks

**Pull request closed without merging**